content/nomos/updates/2024-03-04.md

```diff
@@ -0,0 +1,71 @@
+
+### `development`
+
+- Mixnet network backend skeleton - [PR](https://github.com/logos-co/nomos-node/pull/586).
+- Libp2p stream read/write - [PR](https://github.com/logos-co/nomos-node/pull/587).
+- Emitting packets from mixclient using libp2p stream - [PR](https://github.com/logos-co/nomos-node/pull/588).
+- Handle outputs from mixnode using libp2p stream/gossipsub - [PR](https://github.com/logos-co/nomos-node/pull/589).
+- Refactor Poisson distribution implementation - [PR](https://github.com/logos-co/nomos-node/pull/590).
+- Mix client Poisson emission - [PR](https://github.com/logos-co/nomos-node/pull/591).
+- Mix node packet handling - [PR](https://github.com/logos-co/nomos-node/pull/592).
+- Mix Packet / Fragment logic - [PR](https://github.com/logos-co/nomos-node/pull/593).
+- Move FisherYates to nomos-utils - [PR](https://github.com/logos-co/nomos-node/pull/594).
+- Mixnet topology - [PR](https://github.com/logos-co/nomos-node/pull/595).
+- Mix client/node unit tests [PR](https://github.com/logos-co/nomos-node/pull/596).
+- Note on the PRs above: tests will fail because the whole implementation has been split into small PRs. All tests will pass at the last PR that will be opened.
+- Taking some time to refactor network adapter codes that are tightly coupled with only the libp2p network backend.
+
+## `testnet`
+
+### `development`
+
+- WIP: Integration for explorer - [PR](https://github.com/logos-co/nomos-node/pull/603) (marked as WIP since one of the unit tests is failing).
+
+## `cryptarchia`
+
+### `research`
+
+- [Tokenomics design canvas](https://www.notion.so/Nomos-Tokenomics-Design-Canvas-e62e78ee0f8c4a719d1f6ffb08574241) has been updated with additional clarifications regarding the "Light node" role (both Zone verifiers and providers of replication).
+- [Stake relativization](https://www.notion.so/Stake-Relativization-65f5ed5522b64c36b625652023318d88#1c0df180a7dc45108c2f4aa6a8a256f2) has been updated as part of the Cryptarchia specification.
+- Started the Block rewards [document](https://www.notion.so/Block-rewards-01d08f7a371e4b348ec022e6416e991f) to discuss and propose solutions to rewarding new block proposals.
+
+### `development`
+
+- Merged all PRs for Cryptarchia [here](https://github.com/logos-co/nomos-node/pull/600) and [here](https://github.com/logos-co/nomos-node/pull/583).
+
+## `data availability`
+
+### `research`
+
+- The examination of the Taiga design continues. Technical details and the cryptographic functions used are being researched. Verifiable encryption is under consideration, and existing libraries in the literature are being reviewed. Notes and questions related to the Taiga design are documented [here](https://chat.openai.com/c/806109a9-202b-49e5-91dc-ffd2bd76c43f) (WIP).
```

Comment:
Taiga is Coordination Layer. This should go in a new section, since we started with this.